<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QWidget" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>443</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thay đổi mật khẩu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/icon-file/resetpassword.png</normaloff>:/icon-file/resetpassword.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(131, 197, 197);
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">font : 8pt Courier;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Mật khẩu cũ :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Mật khẩu mới :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Nhập lại mật khẩu mới :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="styleSheet">
           <string notr="true">QLineEdit
{
	background-color: none;
	padding-left : 10px;
	border-radius : 4px;
	border : 1px solid;
}
QLineEdit:hover
{
	border : 1px solid rgb(255, 170, 0);
}
QLineEdit:focus
{
	border : 1px solid rgb(255, 170, 0);
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_oldpass">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>167</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);};
</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_newpass">
             <property name="minimumSize">
              <size>
               <width>167</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);};
</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_newpassres">
             <property name="minimumSize">
              <size>
               <width>167</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{background-color: rgb(255, 255, 255);};
</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>27</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color :white;
	border:2px groove rgb(255, 170, 0);
	border-style : groove;
	border-width : 2px;
	border-radius : 6px;
	padding : 2px;
	font : 8pt Courier;
}
QPushButton:focus
{
	background-color: rgb(255, 170, 0);
	border:4px groove rgb(162, 108, 0);
	outline : 0;
}
QPushButton:hover
{
	background-color: rgb(255, 170, 0);
	border:2px groove rgb(68, 68, 68);
}
QPushButton:pressed
{
	background-color: rgb(255, 170, 0);
	border:1px groove rgb(68, 68, 68);
}
</string>
     </property>
     <property name="text">
      <string>Xác nhận</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
